hibernate-reactive-core/src/test/java/org/hibernate/reactive/configuration/ReactiveConnectionPoolTest.java

```diff
@@ -137,4 +129,18 @@ private void verifyConnectivity(TestContext context, ReactiveConnectionPool reac
 						} ) ) );
 	}
 
+	private void verifyFailedConnectivity(TestContext context, ReactiveConnectionPool reactivePool) {
```

Comment:
`verifyFailedConnectivity` is basically a copy of `verifyConnectivity`.
Instead of duplicating the code, you could change the signature to:
```
CompletionStage<Void> verifyConnectivity(...)
```

than you can have:
```
test(context, verifyConnectivity(...));
```
or
```
test(context, assertThrown( Exception.class, verifyConnectivity(...).thenAccept...);
```

Reply:
A separate method was needed to wrap the state in `ReactiveAssertions.assertThrown()`

Reply:
@blafond, this is what I meant: https://github.com/hibernate/hibernate-reactive/blob/793815dd9ab9148f50fb69604a77a63b7290ad10/hibernate-reactive-core/src/test/java/org/hibernate/reactive/configuration/ReactiveConnectionPoolTest.java#L124

No need to have duplicated code.